app/views/curation_concerns/base/_show_actions.html.erb

```diff
@@ -1,8 +1,4 @@
-<%# TODO: set these variables based on the current user
-  # See https://github.com/projecthydra/sufia/issues/1477 %>
-<% collector = true %>
-<% editor = true %>
-<% if collector || editor %>
+<% if @presenter.collector || @presenter.editor %>
```

Comment:
Perhaps leave the collector role for another PR as suggested by @mjgiarlo.  Permitting analytics and citations to be viewable by anyone with permissions to view the object seems reasonable to me.